src/sentry/monitors/models.py

```diff
@@ -251,7 +251,7 @@ class Monitor(Model):
     Type of monitor. Currently there are only CRON_JOB monitors.
     """
 
-    owner_user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="CASCADE")
+    owner_user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="SET_NULL")
```

Comment:
Nit: I think maybe it's better to use `HybridCloudForeignKeyCascadeBehavior.SET_NULL`. Both are valid, but it's easier to see all the options for other people looking for examples

Reply:
Yeah but we don't do it anywhere else lol

Reply:
🥲

Reply:
let me clean up